widget/tabcontainer.go

```diff
@@ -163,10 +144,42 @@ func (t *TabContainer) buildTabBar(buttons []fyne.CanvasObject) *fyne.Container
 	return tabBar
 }
 
+// Append adds a new CanvasObject to the end of the group
```

Comment:
This looks like a copy paste issue. The docs seem slightly terse but apart from this line they are correct ;)